README.md

```diff
@@ -15,13 +15,6 @@ You will also need to install the following on your development machine:
 - Go to project root, then run `npm install`
 - Grunt (`sudo npm install -g grunt-cli`)
 
-
```

Comment:
I figured I could remove it since it's already mentioned above?
```
You will also need to install the following on your development machine:

Ruby, RubyGems
Sass (sudo gem install sass)
Node.js and NPM (see https://nodejs.org/en/download/package-manager/)
Go to project root, then run npm install
Grunt (sudo npm install -g grunt-cli)
```

Reply:
True. let's remove it then